src/app/pages/datasets/components/dataset-capacity-management-card/dataset-capacity-management-card.component.html

```diff
@@ -1,42 +1,104 @@
+
+    <div class="chart-extra">
+      <div fxFlex="row" fxLayoutAlign="space-around center">
+        <div class="details-item" *ngIf="!isZvol">
+          <span class="label">{{ 'Reserved for Dataset' | translate }}:</span>
+          <span class="value" *ngIf="!isLoading; else loaderSkel">
+            <!-- TODO: Show real value -->
+            {{ extraProperties.reservation.parsed | filesize: { standard: 'iec', round: 0 } }}
+          </span>
+        </div>
+        <div class="details-item" *ngIf="!isZvol">
+          <span class="label">{{ 'Reserved for Dataset & Children' | translate }}:</span>
+          <span class="value" *ngIf="!isLoading; else loaderSkel">
+            <!-- TODO: Show real value -->
+            {{ extraProperties.refreservation.parsed | filesize: { standard: 'iec', round: 0 } }}
+          </span>
+        </div>
+        <div class="details-item" *ngIf="isZvol">
+          <span class="label">{{ 'Provisioning Type' | translate }}:</span>
+          <!-- TODO: Show real value -->
+          <span class="value" *ngIf="!isLoading; else loaderSkel">{{ 'Thick' | translate }}</span>
+        </div>
+        <div class="details-item" *ngIf="isZvol">
+          <span class="label">{{ 'Volume Size' | translate }}:</span>
+          <span class="value" *ngIf="!isLoading; else loaderSkel">
+            {{ extraProperties?.volsize?.parsed | filesize: { standard: 'iec', round: 0 } }}
+          </span>
+        </div>
+      </div>
     </div>
-    <div class="legend-placeholder">
-      <ng-container *ngIf="loading; else loaderSkel"></ng-container>
-      <ng-container *ngIf="loading; else loaderSkel"></ng-container>
-      <ng-container *ngIf="loading; else loaderSkel"></ng-container>
+
+    <div class="details">
+      <div class="details-item">
+        <span class="label">
+          <ng-container *ngIf="isFilesystem">{{ 'Space Available to Dataset' | translate }}</ng-container>
+          <ng-container *ngIf="isZvol">{{ 'Space Available to Zvol' | translate }}</ng-container>:
+        </span>
+        <span class="value" *ngIf="!isLoading; else loaderSkel">
+          {{ dataset.available.parsed | filesize: { standard: 'iec', round: 0 } }}
+        </span>
+      </div>
+      <div class="details-item" *ngIf="isFilesystem">
+        <span class="label">
+          {{ 'Applied Dataset Quotas' | translate }}:
+        </span>
+        <ng-container *ngIf="checkQuotas; else notAvailable">
+          <span class="value" *ngIf="!isLoading; else loaderSkel">
+            {{ appliedQuotas | filesize: { standard: 'iec', round: 0 } }}
+          </span>
+        </ng-container>
+      </div>
+      <div class="details-item">
+        <span class="label">
+          {{ 'Inherited Quotas' | translate }}:
+        </span>
+        <ng-container *ngIf="checkQuotas; else notAvailable">
+          <span class="value" *ngIf="!isLoading; else loaderSkel">
+            {{ inheritedQuotas | filesize: { standard: 'iec', round: 0 } }}
```

Comment:
I think it should be something like 'Inherited Quotas: 50 GiB from my-dataset' and the link on the right should say 'Go to my dataset'